### ISSUE TITLE
Navigatable Grid cannot be sorted or filtered with tap

### ISSUE THREAD

**Issue by VicTachev** [telerik/kendo-ui-core#4524]
Taping on the Grid header does not sort the column on touch devices. The problem is first observed in version 2018.2.516.

### Reproduction of the problem
1. Open [this dojo](http://runner.telerik.io/fullscreen/aZOnAhUZ/2)
2. Enable mobile emulation in Chrome DevTools
3. Click/tap on a column header to sort or filter
 
### Current behavior
The column is not sorted/filter menu is not opened

### Expected/desired behavior
The column should be sorted/filter menu should open

### Environment

* **Kendo UI version:** 2018.2.516 and later
